demos/whisper-base/whisper.js

```diff
@@ -78,14 +78,31 @@ export class Whisper {
                 title: "Whisper Base Decoder (Cached)",
             },
         };
+        this.hidden_state_shape = [1, 1500, 512];
         this.kv_encoder_shape = [1, 8, 1500, 64];
         this.kv_decoder_shape = [1, 8, 127, 64];
         this.first_logits_shape = [1, 4, 51865];
         this.kv_logits_shape = [1, 1, 51865];
 
         // Pre-allocated TypedArrays for IO binding reuse
+        this.logits_buffer = null;
         this.kv_logits_buffer = null;
 
+        // Pre-allocated MLTensors for IO binding (initialized once)
```

Comment:
The `pre` prefix (like anti, dis, hyper) affixes itself to the base word (like [preabdomen, preablation, preadmit](https://en.wiktionary.org/wiki/preallocate)...)

```suggestion
        // Set up preallocated MLTensors for IO binding (initialized once).
```

Below too.